scripts/fast-reboot

```diff
@@ -417,6 +417,24 @@ debug "Stopped  bgp ..."
 docker kill lldp > /dev/null
 systemctl stop lldp
 
+# Kill iccpd
+#docker kill iccpd > /dev/null
+
+if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
+    docker kill iccpd > /dev/null
```

Comment:
what if iccpd docker is not in the image.

Reply:
yes，we need to ensure docker iccpd is here.